docker-compose.test.yml

```diff
@@ -18,7 +18,7 @@ services:
       - BB_SERVER_LOCATION=https://fhir.backend.bluebutton.hhsdevcloud.us
       - BB_CHECK_CERT=false
       - FHIR_PAYLOAD_DIR=../bcdaworker/data
-      - FHIR_STAGING_DIR=../bcdaworker/tmpdata
+      - FHIR_STAGING_DIR=data/test
```

Comment:
I know we talked about this being the ultimate solution going forward, but I thought this was going to be cleaned up in a tech debt ticket?  Making this singular change in the compose stack could have integration test repercussions, as well as creates a sync issue with the other env vars that are pointing to `/bcdaworker/data`.  Can we be consistent and stick with modifying `FHIR_STAGING_DIR` in the test setup: https://github.com/CMSgov/bcda-app/blob/knollfear/BCDA-1233/bcdaworker/main_test.go#L38-L46
And then just create a ticket to coordinate/sync the compose stack with the internal unit test usage.
Note, I don't think any of this affects the production environment variables in S3.

Reply:
The tests pass so this is a workable fix.  I don't understand the objection to this.  We can still write a ticket to do something else but doing it this way isn't going to change anything about our ability to do somethign else in the future.  Furthermore, this is just a kludge to fix the fact that SonarCube should just ignore this folder. I actually don't think we should ever be modifying the fhir staging dir in any tests unless that is specifically part of the test.  We don't flip flop that dir in production, we shouldn't be doing it in tests unless the test demands it.

Reply:
Yeah, I wonder why the tests were updated to use that separate FHIR_STAGING_DIR in the first place?  Not sure if Sonar is to blame for that one, but ultimately we should investigate to see if Sonar has a .ignore file we can leverage.

Reply:
Those vars are also only set that way because the relative paths from the docker-compose file are not valid because the test runs from a fundamentally different path.  The app makes its own download directories on boot so setting it in docker compose should work regardless of the execution point (real vs tests)

Reply:
We can use `sonar.exclusions` to ignore that directory: https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/